specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2022-11-01/keysManagedHsm.json

```diff
@@ -0,0 +1,624 @@
+        "exp": {
+          "x-ms-client-name": "Expires",
+          "type": "integer",
+          "format": "int64",
+          "description": "Expiry date in seconds since 1970-01-01T00:00:00Z."
+        },
+        "created": {
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true,
+          "description": "Creation time in seconds since 1970-01-01T00:00:00Z."
+        },
+        "updated": {
+          "type": "integer",
+          "format": "int64",
+          "readOnly": true,
+          "description": "Last updated time in seconds since 1970-01-01T00:00:00Z."
+        },
+        "recoveryLevel": {
+          "type": "string",
+          "description": "The deletion recovery level currently in effect for the object. If it contains 'Purgeable', then the object can be permanently deleted by a privileged user; otherwise, only the system can purge the object at the end of the retention interval.",
+          "enum": [
+            "Purgeable",
+            "Recoverable+Purgeable",
+            "Recoverable",
+            "Recoverable+ProtectedSubscription"
+          ],
+          "x-ms-enum": {
+            "name": "DeletionRecoveryLevel",
+            "modelAsString": true
+          },
+          "readOnly": true,
+          "x-nullable": false
+        },
+        "exportable": {
+          "type": "boolean",
+          "description": "Indicates if the private key can be exported."
+        }
+      },
+      "description": "The object attributes managed by the Azure Key Vault service.",
+      "type": "object"
+    },
+    "ManagedHsmKeyCreateParameters": {
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "The tags that will be assigned to the key."
+        },
+        "properties": {
+          "$ref": "#/definitions/ManagedHsmKeyProperties",
+          "description": "The properties of the key to be created."
+        }
+      },
+      "description": "The parameters used to create a key.",
+      "required": [
+        "properties"
+      ],
+      "x-ms-azure-resource": true,
+      "type": "object"
+    },
+    "ManagedHsmKey": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/ManagedHsmKeyProperties",
+          "description": "The properties of the key."
+        }
+      },
+      "required": [
+        "properties"
+      ],
+      "allOf": [
+        {
+          "$ref": "./managedHsm.json#/definitions/ManagedHsmResource"
+        }
+      ],
+      "description": "The key resource.",
+      "type": "object"
+    },
+    "ManagedHsmKeyListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagedHsmKey"
+          },
+          "description": "The key resources."
+        },
+        "nextLink": {
```

Comment:
>nextLink [](http://example.com/codeflow?start=9&length=8)

nit can add format uri

Reply:
I see this same pattern in most of the places. What is the expected change here?